downstream/modules/hub/proc-upload-collection.adoc

```diff
@@ -13,7 +13,7 @@
 To upload your collection using the Automation Hub user interface:
 
 . Log in to Red Hat Ansible Automation platform.
-. Navigate to menu:Automation Hub[My Namespaces].
+. From the navigation panel, select menu:Automation Hub[My Namespaces].
```

Comment:
My Namespaces should be Namespaces.

Reply:
Hi @dcdacosta. I updated the procedure, per your information above. Please confirm that I added them correctly. I was unsure if the final steps should remain (steps 17-19). I have removed only 17 and 18 for now.